### DISCUSSION TITLE
Verbindungsprobleme

### DISCUSSION THREAD

**Discussion by Teddy-dirk** [community/community#133249] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone, I'm relatively new to coding and still very inexperienced.
Well, I have created repositories on a trial basis. So far so good. My Html.code is also provided in the browser. However, without my resources. Everything works fine locally. But unfortunately my attachments are not used via GitHub. (img, video and style.css)
Can someone please help me? https://github.com/Teddy-dirk/Teddy-dirk.github.io.-excursion

**Comment by queenofcorgis** [comment]
Hi @Teddy-dirk , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)] very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
